### ISSUE TITLE
[a11y]: Search - Expandable With Layer - For all layers , JAWS is announcing as Layer 1

### ISSUE THREAD

**Issue by 2nikhiltom** [carbon-design-system/carbon#16636]
### Package

@carbon/react

### Browser

Safari

### Package version

v1.58.0

### Automated testing tool and ruleset

Manual QA

### Assistive technology

JAWS

### Description

In Tab key navigation, on expanding the search button of Layer 2 and layer 3, JAWS is announcing as "_First Layer Edit blank Placeholder Search Type in text_".

In arrow key navigation, Without expanding the search button of Layer 2 and Layer 3, JAWS is announcing as "_First Layer Edit blank Placeholder Search Type in text_".

### Reproduction/example

https://react.carbondesignsystem.com/?path=/story/components-search--expandable-with-layer

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by ibm-sint** [comment]
@tay1orjones I think this is a similar issue to something our client is facing and they are saying it is a violation of WCAG WCAG 1.3.1 (Info and Relationships) and/or 4.1.2 (Name, Role, Value). Are there any updates on it?

I have recreated it with VoiceOver on the Carbon website, so this is not just a JAWS issue.

https://github.com/user-attachments/assets/3d090e9d-e934-4131-8d47-4272e4c78065

**Comment by tay1orjones** [comment]
I ran this by the IBM Accessibility team, and while we can't identify a specific WCAG criteria that's violated in this scenario, we agree that there is a user experience concern for some users of assistive technologies that we'll get fixed. Specifically, the transparent content should be `visibility:hidden`.

It's a somewhat common case of "by the letter of the law", there's not a problem. But, by the spirit of the law, it's broken.

* [originating slack convo](https://ibm-studios.slack.com/archives/C2K6RFJ1G/p1779358639612709)
* [slack convo 2](https://ibm-studios.slack.com/archives/G01GCBCGTPV/p1780323828937329)
